### ISSUE TITLE
No Training Configuration Found warning

### ISSUE THREAD

**Issue by ben-heil** [greenelab/simulate-expression-compendia#40]
Divneet is working on running the [sample level simulation](https://github.com/greenelab/simulate-expression-compendia/blob/master/Pseudomonas/Pseudomonas_sample_lvl_sim.ipynb) notebook, and he's running into an issue with model loading. When he runs the simulation on the trained model, keras throws a 'No training configuration found in save file: the model was *not* compiled' warning and doesn't return any results.

The bug doesn't have a stack trace with it, but after doing some digging I believe the relevant functions are:
[main_notebook](https://github.com/greenelab/simulate-expression-compendia/blob/master/Pseudomonas/Pseudomonas_sample_lvl_sim.ipynb) 
-> [pipeline.run_simulation](https://github.com/greenelab/simulate-expression-compendia/blob/master/simulate_expression_compendia_modules/pipeline.py#L51) 
-> [simulations.sample_level_simulation](https://github.com/greenelab/simulate-expression-compendia/blob/6b03bcfe17c01699f6ec0868e6696644870bac36/simulate_expression_compendia_modules/simulations.py#L28) 
-> [simulate_expression_data.simulate_by_random_sampling](https://github.com/greenelab/ponyo/blob/09096eada5f31ab5c28a2e743fd0539d432ca7a7/ponyo/simulate_expression_data.py#L69) 
-> [loaded_model = load_model(model_encoder_filename)](https://github.com/greenelab/ponyo/blob/09096eada5f31ab5c28a2e743fd0539d432ca7a7/ponyo/simulate_expression_data.py#L145)

The model being loaded is saved [in this script](https://github.com/greenelab/ponyo/blob/09096eada5f31ab5c28a2e743fd0539d432ca7a7/ponyo/vae.py#L306).

In my past I ran into similar issues due to using custom loss functions and forgetting to save them along with the function. Have you run into this issue before?

**Comment by ajlee21** [comment]
I have not seen this error before.

Just to clarify, this error is thrown at this step?
```
pipeline.run_simulation(config_file,
                        normalized_processed_data_file,
                        corrected=False)
```
The encoder model should all be stored in `https://github.com/greenelab/simulate-expression-compendia/tree/master/Pseudomonas/models/NN_2500_30` 

They are git lfs tracked..I'm not sure if that is what is causing the issue..? Happy to come to the meeting this week to help debug this if that will help too

**Comment by ben-heil** [comment]
Hmm weird. Yeah, can you come to the meeting tonight? I'll slack you an invite

**Comment by ben-heil** [comment]
Update: likely caused by training getting skipped by a line checking whether anything is in the log file. New error: the config file is missing a `validation_frac` parameter

**Comment by ajlee21** [comment]
Made a new issue for the config file: https://github.com/greenelab/simulate-expression-compendia/issues/41
